pkg/host/internal/sriov/sriov.go

```diff
@@ -189,12 +189,13 @@ func (s *sriov) VFIsReady(pciAddr string) (netlink.Link, error) {
 	var err error
 	var vfLink netlink.Link
 	err = wait.PollImmediate(time.Second, 10*time.Second, func() (bool, error) {
-		vfName := s.networkHelper.TryGetInterfaceName(pciAddr)
-		vfLink, err = s.netlinkLib.LinkByName(vfName)
+		vfIndex := s.networkHelper.TryGetInterfaceIndex(pciAddr)
```

Comment:
Do we want to explicitly handle -1 result?

Reply:
we didn't handle the empty interface name on TryGetInterfaceName so I was thinking it could be the same but I don't really care :)

Reply:
done